website/docs/error-reference.md

```diff
@@ -1093,6 +1093,31 @@ Sorbet allows the second `if` because we've explicitly made `x` unanalyzable
 with `T.unsafe(...)`. T.unsafe is one of a handful of
 [escape hatches](troubleshooting.md#escape-hatches) built into Sorbet.
 
+## 7007
+
+Sorbet found a mistmatch between the type specified for a parameter in the
+signature and the default value provided.
+
+In this case, the signature states that `category` type is a `Category`, yet we
+try to use `nil` as default value:
```

Comment:
It just so happens that default argument type checking reports this error by nature of how Sorbet desugars default arguments. This error code can come from hand-written `T.let` calls too:

https://sorbet.run/#%23%20typed%3A%20true%0Aextend%20T%3A%3ASig%0A%0Ax%20%3D%20T.let(1%2C%20String)%0A%0AMyCnst%20%3D%20T.let(1%2C%20String)

Do you mind mentioning this?

Reply:
Oh I see, here's the updated version:

![image](https://user-images.githubusercontent.com/583144/127556881-c9fe5800-9fa5-425f-be04-70e55045ac33.png)